stack-ghc881.yaml

```diff
@@ -0,0 +1,3 @@
+resolver: nightly-2020-01-21
```

Comment:
Could you add a comment here describing why `lts-15.0` doesn't work? I think that as soon as circumstances change, we'll want to switch to `lts-15.0` and a comment describing the current issue will be useful then.